api/tests/integration/infrastructure/repositories/user-repository_test.js

```diff
@@ -479,90 +480,130 @@ describe('Integration | Infrastructure | Repository | UserRepository', () => {
+        expect(firstSchoolingRegistration).to.be.instanceOf(SchoolingRegistrationForAdmin);
+        expect(firstSchoolingRegistration.firstName).to.equal(firstSchoolingRegistrationInDB.firstName);
+        expect(firstSchoolingRegistration.lastName).to.equal(firstSchoolingRegistrationInDB.lastName);
+        expect(firstSchoolingRegistration.birthdate).to.equal(firstSchoolingRegistrationInDB.birthdate);
+        expect(firstSchoolingRegistration.division).to.equal(firstSchoolingRegistrationInDB.division);
+        expect(firstSchoolingRegistration.organizationId).to.equal(firstOrganizationInDB.id);
+        expect(firstSchoolingRegistration.organizationExternalId).to.equal(firstOrganizationInDB.externalId);
+        expect(firstSchoolingRegistration.organizationName).to.equal(firstOrganizationInDB.name);
+        expect(firstSchoolingRegistration.createdAt).to.deep.equal(firstSchoolingRegistrationInDB.createdAt);
+        expect(firstSchoolingRegistration.updatedAt).to.deep.equal(firstSchoolingRegistrationInDB.updatedAt);
+
+        const secondSchoolingRegistration = userDetailsForAdmin.schoolingRegistrations[1];
+        expect(secondSchoolingRegistration).to.be.instanceOf(SchoolingRegistrationForAdmin);
+        expect(secondSchoolingRegistration.firstName).to.equal(secondSchoolingRegistrationInDB.firstName);
+        expect(secondSchoolingRegistration.lastName).to.equal(secondSchoolingRegistrationInDB.lastName);
+        expect(secondSchoolingRegistration.birthdate).to.equal(secondSchoolingRegistrationInDB.birthdate);
+        expect(secondSchoolingRegistration.division).to.equal(secondSchoolingRegistrationInDB.division);
+        expect(secondSchoolingRegistration.organizationId).to.equal(secondOrganizationInDB.id);
+        expect(secondSchoolingRegistration.organizationExternalId).to.equal(secondOrganizationInDB.externalId);
+        expect(secondSchoolingRegistration.organizationName).to.equal(secondOrganizationInDB.name);
+        expect(secondSchoolingRegistration.createdAt).to.deep.equal(secondSchoolingRegistrationInDB.createdAt);
+        expect(secondSchoolingRegistration.updatedAt).to.deep.equal(secondSchoolingRegistrationInDB.updatedAt);
       });
-
     });
 
-    context('when user is not authenticated from GAR', () => {
-
-      let userInDb;
-      const userNotAuthenticatedFromGAR = {
-        firstName: faker.name.firstName(),
-        lastName: faker.name.lastName(),
-        email: faker.internet.exampleEmail().toLowerCase(),
-        password: bcrypt.hashSync('A124B2C3#!', 1),
-        cgu: true,
-        samlId: null,
-      };
+    context('when user has schoolingRegistrations from not SCO organization', () => {
 
-      beforeEach(() => {
-        userInDb = databaseBuilder.factory.buildUser(userNotAuthenticatedFromGAR);
-        return databaseBuilder.commit();
-      });
+      it('should return the user with his schoolingRegistrations', async () => {
```

Comment:
Si l'attendu est le même que celui du précédent contexte, pourquoi séparer les deux ici ?

Reply:
Je viens de relire, on s'attend en fait à ce qu'il n'y ait pas de schoolingRegistrations dans ce cas

Reply:
Du coup, je propose un changement pour que l'attendu soit un peu plus clair :
```suggestion
      it('should return the user with empty schoolingRegistrations', async () => {
```

Reply:
Done